src/librfu_rfu.c

```diff
@@ -1426,7 +1430,11 @@ static u16 rfu_STC_setSendData_org(u8 ni_or_uni, u8 bmSendSlot, u8 subFrameSize,
     else if (gRfuLinkStatus->parentChild == MODE_CHILD)
         llFrameSize_p = &gRfuLinkStatus->remainLLFrameSizeChild[bm_slot_id];
     frameSize = llsf_struct[gRfuLinkStatus->parentChild].frameSize;
+#ifdef UBFIX
+    if (llFrameSize_p && (subFrameSize > *llFrameSize_p || subFrameSize <= frameSize))
+#else
     if (subFrameSize > *llFrameSize_p || subFrameSize <= frameSize)
+#endif
```

Comment:
Technically this should be
```c
if ((llFrameSize_p && subFrameSize > *llFrameSize_p) || subFrameSize <= frameSize)
```
because you still want to check the latter condition if `llFrameSize_p` is null (I couldn't tell you if this matters, I don't know if this might ever be called when `parentChild` is `MODE_NEUTRAL`).